requirements.txt

```diff
@@ -21,12 +20,11 @@ pathlib
 WeasyPrint==0.42.3
 xlsxwriter
 click<7.0
-requests-oauthlib<1.2.0
-oauthlib<3.0.0
-flask_ldap3_login
 
-# unclear
-Flask-Login
+# webapp login
+authlib
+flask_ldap3_login
+flask_login
```

Comment:
This lib stays, it's the one that is logging the user (regardless if it's obtained from google, ldap or email) in the app

Reply:
Really nice to be able to clean those 😄